vsts/pipelines/buildimage-bases.yml

```diff
@@ -15,12 +15,12 @@ trigger:
     - images/build/php
     - images/build/yarn-cache
     - build/
-    - vsts/
```

Comment:
why remove vsts from the list? everything that impacts the build should be in the list, right?

Reply:
I reverted this check for Arjun

Reply:
Thanks @kichalla